core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/ambari/AmbariLdapService.java

```diff
@@ -58,6 +59,9 @@ public void setupLdap(Stack stack, Cluster cluster, AmbariRepo ambariRepo) {
             ldapConfigs.put("ambari.ldap.connectivity.anonymous_bind", false);
             if (ambariRepositoryVersionService.isVersionNewerOrEqualThanLimited(ambariRepo::getVersion, AMBARI_VERSION_2_7_2_0)) {
```

Comment:
I know that this is not a new code but I guess instead of "convert" it should be "skip" and I am not sure why we need this version check at the front of it

Reply:
According to @sodre90 the fix is introduced in that version, it is throwing the same error otherwise. (https://hortonworks.jira.com/browse/BUG-112927). The value can be skip, but it should be backported to the property file based method then, before Ambari 2.7, so I would do it in a different issue.